index.html

```diff
@@ -74,6 +74,7 @@
     <script src="thirdPartyCode/toolsocket/index.js"></script>
     <script src="thirdPartyCode/hull.js"></script>
     <script src="thirdPartyCode/stats.min.js"></script>
+    <script src="thirdPartyCode/three-mesh-bvh.module.js" type="module"></script>
```

Comment:
This line shouldn't be necessary since threejsScene imports three-mesh-bvh as a library. Having the script tag will run the code in three-mesh-bvh as if it were a standalone thing

Reply:
Thanks for the correction!